test/auth.test.ts

```diff
@@ -0,0 +1,57 @@
+import { setupRecording, setupAPI } from "../test/setup";
+import { SaleorSDK } from "../src/core";
+import { TEST_AUTH_EMAIL, TEST_AUTH_PASSWORD } from "../config";
+
+describe("auth api", () => {
+  const context = setupRecording();
+  const { client } = setupAPI();
+  const saleor = SaleorSDK(client);
+
+  beforeEach(() => {
+    const { server } = context.polly;
+    server.any().on("beforePersist", (_, recording) => {
+      const requestJson = JSON.parse(recording.request.postData.text);
+      const responseHeaders = recording.response.headers.filter(
+        (el: any) => el.name !== "set-cookie"
+      );
+
+      delete requestJson.variables?.email;
+      delete requestJson.variables?.password;
+
+      recording.request = {
+        ...recording.request,
+        postData: {
+          ...recording.request.postData,
+          text: JSON.stringify(requestJson),
+        },
+      };
+      recording.response = {
+        ...recording.response,
+        cookies: [],
+        headers: responseHeaders,
+      };
```

Comment:
Since we're mutating these anyway, it would be more readable to do it directly over recreating the whole object:
```suggestion
      recording.request.postData.text = text: JSON.stringify(requestJson);
      recording.response.cookies =  [];
      recording.response.headers = responseHeaders;
```